chapter_12_organising_test_files.asciidoc

```diff
@@ -466,7 +467,6 @@ might decide that building a specific helper method is overkill at this stage,
 but it might be nice to have some generic way of saying, in our tests, "wait
 until this assertion passes".  Something like this:
 
-// DAVID: might be worth reminding the reader about that helper method, e.g. what it's called.
```

Comment:
in my opinion it isn't the focus here (I mean we just merely mention this other method)